docs/src/pages/docs/crocks/Writer.md

```diff
@@ -0,0 +1,352 @@
+
+```javascript
+import Writer from 'crocks/Writer'
+
+// StringWriter :: Writer String a
+const StringWriter =  Writer(String)
+
+StringWriter('zero', 0)
+// Writer String Number
+
+StringWriter('adding inputs', (a, b) => a + b)
+// Writer String Number
+```
+
+</article>
+
+<article id="topic-constructor">
+
+## Constructor Methods
+
+#### of 
+
+```haskell
+Writer.of :: a -> Writer m a
+```
+
+`of` is a construction helper that can be used to construct an `Writer` with any given value, with an empty instance of the `Monoid` that was passed to the type constructor.
+
+```javascript
+import Writer from 'crocks/Writer'
+
+import List from 'crocks/List'
+
+// StringWriter :: Writer List a
+const ListWriter =  Writer(List)
+
+ListWriter.of(42)
+// Writer List Number
+```
+
+</article>
+
+<article id="topic-instance">
+
+## Instance Methods
+
+#### equals
+
+```haskell
+Writer m a ~> b -> Boolean
+```
+
+Used to compare the underlying values of two `Writer` instances for equality by value, `equals` takes any given argument and returns `true` if the passed arguments are a `Writer` with an underlying value are equal to the underlying value of the `Writer` the method is being called on. If the passed argument is not a `Writer` instance or if the underlying values are not equal, `equals` will return false.
+
+```javascript
+import Assign from 'crocks/Assign'
+import Writer from 'crocks/Writer'
+
+import equals from 'crocks/pointfree/equals'
+
+// AssignWriter :: Writer Assign a
+const AssignWriter =  Writer(Assign)
+
+AssignWriter.of(2)
+  .equals(AssignWriter.of(2))
+//=> true
+
+AssignWriter(Assign({ a: 20 }), null)
+  .equals(AssignWriter(Assign({ a: 30 }), null))
+//=> true
+
+Writer(String)('here is 42', { value: 42 })
+  .equals(Writer(String)('also 42', { value: 42 }))
+//=> true
+
+Writer(String)('hello', 25)
+  .equals(Writer(String)('hello', 26))
+//=> false
+
+equals(Writer(Array).of('result'), 'result')
+//=> false
+```
+
+#### map
+
+Used to apply transformations to the value that a `Writer` holds, `map` takes a function that it will lift into the context of the `Writer` and apply to it the wrapped value. 
+
+```javascript
+import Sum from 'crocks/Sum'
+import Writer from 'crocks/Writer'
+
+// toLower :: String -> String
+const toLower = x => x.toLowerCase()
+
+Writer(Sum).of('HELLO!')
+  .map(toLower)
+//=> Writer( Sum 0 "hello!" )
+
+Writer(String)('file was loaded', 'important text from document')
+  .map(contents => contents.length)
+//=> Writer("file was loaded"  28)
+```
+
+#### ap
+
+Short for apply, `ap` is used to apply a `Writer` instance containing a value to another `Writer` instance that contains a function, resulting in new `Writer` instance comprising of the result of concatenating the monoid instances, and the value derived from applying the function. `ap` requires that it is called on a writer that wraps a curried polyadic function as its value.
```

Comment:
is polyadic the right word here? i had to look it up so not sure either way, just checking

Reply:
It's been consistently used across other docs and it's descriptive imho.

Reply:
Sounds good to me, i just hadn't seen it